addons/longhorn/template/base/install.sh

```diff
@@ -139,7 +151,7 @@ function maybe_init_hosts() {
         local desired=$(kubectl get daemonsets -n longhorn-system longhorn-manager --no-headers | tr -s ' ' | cut -d ' ' -f2)
         local ready=$(kubectl get daemonsets -n longhorn-system longhorn-manager --no-headers | tr -s ' ' | cut -d ' ' -f4)
 
-        if [ "$desired" = "$ready" ] && [ -n "$desired" ] && [ "$desired" > "1" ]; then
+        if [ "$desired" = "$ready" ] && [ -n "$desired" ] && [ "$desired" -ge "1" ]; then
```

Comment:
This was a shellcheck error.

Reply:
I also don't think it was doing anything before, because the value of "1" is totally valid.